Sources/TSCTestSupport/Product.swift

```diff
@@ -101,6 +101,8 @@ extension Product {
         // Unset the internal env variable that allows skipping certain tests.
         environment["_SWIFTPM_SKIP_TESTS_LIST"] = nil
 
+        environment["DYLD_LIBRARY_PATH"] = nil
```

Comment:
is this a temporary measure? seems like this could lead to issues down the line?

Reply:
Yah, definitely seems temporary to me. This seems to be the underlying issue rdar://75752698